commit 9c5942868c92bb26225ef513876019c9bc10a32b
Author: Chris Leach <cxl000@hotmail.com>
Date:   Mon Nov 25 01:12:27 2013 +0000

    Remove c 11 override keyword
    
    Conflicts:
    	src/shell/animationcurve.h
    	src/shell/desktop-shell.h
    	src/shell/effects/minimizeeffect.h

diff --git a/src/client/element.h b/src/client/element.h
index 2bbd12c..39891c4 100644
--- a/src/client/element.h
+++ b/src/client/element.h
@@ -163,7 +163,7 @@ signals:
 
 protected:
     void setId(int id);
-    virtual void geometryChanged(const QRectF &newGeometry, const QRectF &oldGeometry) override;
+    virtual void geometryChanged(const QRectF &newGeometry, const QRectF &oldGeometry);
 
 private slots:
     void focus(wl_surface *surface, int x, int y);
diff --git a/src/client/layout.h b/src/client/layout.h
index e4c5184..29af40f 100644
--- a/src/client/layout.h
+++ b/src/client/layout.h
@@ -47,9 +47,9 @@ public:
     static LayoutAttached *qmlAttachedProperties(QObject *object);
 
 protected:
-    bool event(QEvent *e) override;
-    void itemChange(ItemChange change, const ItemChangeData &value) override;
-    void geometryChanged(const QRectF &newGeometry, const QRectF &oldGeometry) override;
+    bool event(QEvent *e);
+    void itemChange(ItemChange change, const ItemChangeData &value);
+    void geometryChanged(const QRectF &newGeometry, const QRectF &oldGeometry);
 
 private:
     QList<QQuickItem *> m_items;
diff --git a/src/client/styleitem.h b/src/client/styleitem.h
index 9180243..0e11fc9 100644
--- a/src/client/styleitem.h
+++ b/src/client/styleitem.h
@@ -77,8 +77,8 @@ public:
     void updateLocation(Element::Location loc);
 
 protected:
-    virtual void itemChange(ItemChange change, const ItemChangeData &value) override;
-    virtual void geometryChanged(const QRectF &newGeometry, const QRectF &oldGeometry) override;
+    virtual void itemChange(ItemChange change, const ItemChangeData &value);
+    virtual void geometryChanged(const QRectF &newGeometry, const QRectF &oldGeometry);
 
 signals:
     void itemChanged();
diff --git a/src/client/tooltip.h b/src/client/tooltip.h
index 85b1909..9f69eae 100644
--- a/src/client/tooltip.h
+++ b/src/client/tooltip.h
@@ -40,7 +40,7 @@ public slots:
     void hide();
 
 protected:
-    virtual bool eventFilter(QObject *obj, QEvent *event) override;
+    virtual bool eventFilter(QObject *obj, QEvent *event);
 
 private:
     void doShow();
